### PR TITLE
Update endpoints.adoc

### PR DESCRIPTION
**Pull request by mcherem**
Main Endpoint section with delete procedures.

### CODE REVIEW

**Review by amitfernandes — APPROVED** [review]
Approving "delete endpoints".

File: modules/ROOT/pages/endpoints.adoc [hunk 1]
```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-=======
+// ====
+
+== Delete Endpoints
+
+From either the Partner or Host page, you can delete any endpoints that are not used in message flow configurations.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoints.adoc]: GLOBALLY end a fragment or sentence that leads directly to a graphic, example, list, or step with a colon.
```suggestion
From either the Partner or Host page, you can delete any endpoints that are not used in message flow configurations:
```

File: modules/ROOT/pages/endpoints.adoc [hunk 2]
```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
...
+
+. From within your Anypoint Partner Manager Sandbox or Production environment, select Partner or <host partner name> from the upper left.
+
+. Search for the endpoint you want to delete by Name, Protocol, URL, or Description.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoints.adoc]: Lowercase these as common nouns (you are searching for a name or protocol, for example, and not the label Name or the label Protocol).

Comment (kbostrom) [reply modules/ROOT/pages/endpoints.adoc]:
```suggestion
. Search for the endpoint you want to delete by name, protocol, URL, or description.
```

File: modules/ROOT/pages/endpoints.adoc [hunk 3]
```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
...
++
+You can perform a search from within _Endpoints: Receive from partners_, _Endpoints: Targets at <partner name>_ for the Host, or _Endpoints: Send to <partner name>_ for your partners.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoints.adoc]:
```suggestion
You can search from within _Endpoints: Receive from partners_, _Endpoints: Targets at <partner name>_ for the Host, or _Endpoints: Send to <partner name>_ for your partners.
```

File: modules/ROOT/pages/endpoints.adoc [hunk 4]
```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
...
+
+. If you can’t remember what to search for, scan the list for any endpoints that don’t have a message flow associated with them. These are indicated by a 0 in the message flow column.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoints.adoc]: Limit steps to one sentence max next to the step number. Move other text below.
```suggestion
. If you can’t remember what to search for, scan the list for any endpoints that don’t have a message flow associated with them. 
These are indicated by a 0 in the message flow column.
```
This is a curious step. It is part of the step before it, but it IS a separate user action....
In any case, I am sure the audience will not be confused.

File: modules/ROOT/pages/endpoints.adoc [hunk 5]
```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
...
+
+. Hover over the endpoint name with your mouse until you see the garbage can icon, click it and click *Delete*.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoints.adoc]:
```suggestion
. Hover over the endpoint name until you see the garbage can icon, click the icon, and then click *Delete*.
```

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/endpoints.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoints.adoc]: Fixed.

File: modules/ROOT/pages/endpoints.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoints.adoc]: Fixed.

File: modules/ROOT/pages/endpoints.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoints.adoc]: It's a bad UI design, thus the steps are clunky.

File: modules/ROOT/pages/endpoints.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoints.adoc]: Fixed.

File: modules/ROOT/pages/endpoints.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/endpoints.adoc]: Fixed.

**Pull request merged**